libs/agno/agno/os/interfaces/a2a/router.py

```diff
@@ -35,7 +35,7 @@ def attach_routes(
         raise ValueError("Agents, Teams, or Workflows are required to setup the A2A interface.")
 
     @router.post(
-        "/message/send",
+        "/agents/{agent_id}",
```

Comment:
We shouldn't update the existing endpoints, that would be a breaking change for users that are using this!

- Let's add them as new endpoints. Basically copy/paste the current ones, then update the endpoint and the function name (`run_agent`)
- Let's add a warning in the old endpoints announcing its deprecation. And in sometime, when users have already seen the warning, we will remove them. You can see examples of how to do that warning looking for `warnings.warn` in the repo